hack/run-e2e.sh

```diff
@@ -80,10 +82,21 @@ fi
 
 # build image if not specified
 if [ -z "$PROVISIONER_E2E_IMAGE" ]; then
-    make
-    PROVISIONER_E2E_IMAGE=" k8s.gcr.io/sig-storage/local-volume-provisioner:latest_linux_amd64"
-else
-    docker pull $PROVISIONER_E2E_IMAGE
+    # build and push ${version}_linux_amd64 and ${version}_windows_ltsc2019
+    VERSION=${VERSION:-$(git describe --tags --abbrev=8 --always)}
+    REGISTRY=${REGISTRY:-gcr.io/${PROJECT}}
+    IMAGE=local-volume-provisioner
+
+    REGISTRY=${REGISTRY} \
+        VERSION=$VERSION \
+        CONFIRM=false \
+        ALLOW_UNSTABLE=true \
+        ALLOW_DIRTY=true \
+        LINUX_ARCH="amd64" \
+        WINDOWS_DISTROS="ltsc2019" \
```

Comment:
so we choose ltsc2019 to test, is this configurable?

Reply:
it is configurable but it the test-infra job has to be changed too in, PR for test infra https://github.com/kubernetes/test-infra/pull/24241/files